api/repositories/build_repository.go

```diff
@@ -185,6 +188,22 @@ func (b *BuildRepo) CreateBuild(ctx context.Context, authInfo authorization.Info
 	return b.cfBuildToBuildRecord(cfBuild), nil
 }
 
+func (b *BuildRepo) ListBuilds(ctx context.Context, authInfo authorization.Info) ([]BuildRecord, error) {
```

Comment:
A test should be added in `build_repository_test.go`

Reply:
currently struggling a bit with this. Started adding the test, but somehow I have the feeling my setup of the environment is wrong, as I am ending without any builds when listing builds. Can you maybe my issue here as I am having issues right now finding my mistake.

Reply:
thanks for helping via slack. Adjusted accordingly